### ISSUE TITLE
Firmware 3.0.2. open source

### ISSUE THREAD

**Issue by marcokitt2000** [ANYCUBIC-3D/Kobra2#3]
Hello Anycubic,

There is now 3.0.2. firmware based on marlin bugfix 2.1.x with input shaping and linair advance.
I hope you will release soon this firmware open resource.

By the way when adjust z offset can you set it on the screen 0.05mm step to 0.01 that will be a better way to fine tune z height.
I now there is a pause error (when used m600 for now disabled) some people say its the mainboard others software i think is software. for us pause or filament change or when filament sensor goes off z must goes 10-20 mm up than goes to the front left or right-hand insert  filament but the screen must be updated to for that.
And can we have build in screen manual z height and 4 points and in the middle for height z offset. than a option auto level without cleaning nozzle and calibration module. That it goes bed levelling 25 points. An d a dark screen brightness of lcd auto screen off when touch it goes on etc.

many things to go on.

Thnx for the latest firmware but it can better.

**Comment by Neuroplant** [comment]
Please Fix M600, it is a essential funtion for me

**Comment by neytey1** [comment]
Yes please AnyCubic, release the new 3.0.2 software version's marlin source code!

**Comment by Nazgile94** [comment]
pls opensource! need to enable host actions!
